projects/igniteui-angular/src/lib/grid/cell.component.ts

```diff
@@ -119,26 +303,78 @@ export class IgxGridCellComponent implements OnInit, OnDestroy, AfterViewInit {
         this.cdr.detectChanges();
     }
 
+    /**
+     * Sets/get the `tabindex` property of the cell.
+     * Default value is `0`.
+     * ```typescript
+     * this.cell.tabindex = 1;
+     * ```
+     * ```typescript
+     * let cellTabIndex = this.cell.tabindex;
+     * ```
+     * @memberof IgxGridCellComponent
+     */
     @HostBinding('attr.tabindex')
     public tabindex = 0;
 
+    /**
+     * Sets/get the `role` property of the cell.
+     * Default value is `"gridcell"`.
+     * ```typescript
+     * this.cell.role = 'grid-cell';
+     * ```
+     * ```typescript
+     * let cellRole = this.cell.role;
+     * ```
+     * @memberof IgxGridCellComponent
+     */
     @HostBinding('attr.role')
     public role = 'gridcell';
 
+    /**
+     * Gets whether the cell is editable.
+     * ```typescript
+     * let isCellReadonly = this.cell.readonly;
+     * ```
+     * @memberof IgxGridCellComponent
+     */
     @HostBinding('attr.aria-readonly')
     get readonly(): boolean {
         return !this.column.editable;
     }
 
+    /**
+     * Gets whether the cell is in edit mode.
+     * If `true`, the `"igx_grid__cell--edit"` class is added to the cell.
+     * ```typescript
+     * let cellInEditMode = this.cell.cellInEditMode;
+     * ```
+     * @memberof IgxGridCellComponent
+     */
+    @HostBinding('class.igx_grid__cell--edit')
     get cellInEditMode() {
         return this.inEditMode;
     }
 
+    /**
+     * Gets the `id` of the grid and the `field` of the column in which the cell is stored.
```

Comment:
Returns a string containing the grid `id` and the column `field` concatenated by "_"